python/nano/src/bigdl/nano/automl/pytorch/hposearcher.py

```diff
@@ -133,6 +125,11 @@ def search(self,
             defaults to False.
         :param target_metric: the object metric to optimize,
             defaults to None.
+        :param auto_optimize: Whether to automatically consider the model after
+            inference acceleration in the search process. It will only take
+            effect if target_metric contains "latency". Default value is False.
+        :param input_sample: A set of inputs for trace, defaults to None if you have
+            trace before or model is a LightningModule with any dataloader attached.
```

Comment:
add similar api doc in `Trainer.search` as will

Reply:
Good catch, thx. Done.